lib/vyasa_web/controllers/og_image_controller.ex

```diff
@@ -102,7 +103,15 @@ defmodule VyasaWeb.OgImageController do
     font_size = 70
     caption_text_color = "brown"
 
-    {:ok, base} = Image.open(@img_bg_file_url)
+    bg_url =
+    [:code.priv_dir(:vyasa), "static", "images" ,"logo_with_gradient_and_stamp_1200x630.png"]
+    |> Path.join()
+
+    IO.inspect(@img_bg_file_url, label: "compiletime")
+    IO.inspect(bg_url, label: "runtime")
+
+    {:ok, base} = bg_url
+    |> Image.open()
```

Comment:
(5) NOTE: there was a difference observed between compile and runtime here. 

* compile time: when using the module-level name @priv_dir points to the `_build` directory 
* run time: correctly points to the priv dir

```bash
compiletime: "/app/_build/prod/lib/vyasa/priv/static/images/logo_with_gradient_and_stamp_1200x630.png"
2024-03-16T15:42:28Z app[3d8d9564b91258] sin [info]runtime: "/app/lib/vyasa-0.1.0-alpha.1/priv/static/images/logo_with_gradient_and_stamp_1200x630.png"
```


cursory looks: 
1. seems like using app_dir() is more consistent across runtime vs compile time ([ref forum comment](https://elixirforum.com/t/accessing-the-priv-folder-from-mix-release/24403/7))
2. [this](https://elixirforum.com/t/code-priv-dir-my-app-points-to-priv-directory-in-build/26404/5) is a more root-cause reason, from a core-team member. This thread is useful, seems like it's a matter of release in prod mode vs release in dev mode. 
> ![image](https://github.com/ve1ld/vyasa/assets/38996397/2bc29d7d-aca1-4586-a13e-5265369172bb)